### DISCUSSION TITLE
Astro site does not deploy properly.

### DISCUSSION THREAD

**Discussion by mikhi32** [vercel/community#4723] [Help]
### Summary

Hello everyone! I have a site, built with astro and svelte. It's a mostly photographic site, with a lot of photos. It works fine locally, but when i deploy it to vercel, it gives me problems. 

The site deploys like normal, with no problems in the actual deployment. But when i vist it it's just blank. Using inspect element reveleas there is nothing in the acutal page. Plus, the small preview window actually shows an error.

I really don't know what is the problem. I have checked vercel status and everything is fine (except edge configs, but i am not using that service).

Thanks in advance to anyone.

astro.config.mjs:

```
import { defineConfig } from 'astro/config';
import vercel from '@astrojs/vercel/serverless';

import svelte from "@astrojs/svelte";

// https://astro.build/config
export default defineConfig({
  output: "server",
  adapter: vercel(),
  integrations: [svelte()]
});
```

Screen shots

<img width="2220" alt="Screenshot 2023-11-03 at 11 30 35" src="https://github.com/vercel/community/assets/99438936/6503fb4e-5a08-4d53-b1a5-244b9ba92bf9">

### Steps to Reproduce

1. Deploy the site
2. Wait for deployment
3. Visit the site

**Comment by mikhi32** [comment]
More context, i built locally and for some reason only the page that does not import the svelte component is in the final dist directory.

Not even the layout. Maybe it's a problem with astro?

Here is the .build directory (the same as when it's published to vercel)

 output
   ├── config.json
   ├── functions
   │  └── render.func
   │     ├── dist
   │     │  ├── _empty-middleware.mjs
   │     │  ├── chunks
   │     │  │  ├── 1Caparica _c3e85269.mjs
   │     │  │  ├── 1Catch me if you can_3e248f5e.mjs
   │     │  │  ├── 1Temps primordiaux_35af2280.mjs
   │     │  │  ├── Abbaglio _f681c5c3.mjs
   │     │  │  ├── Alone._ecac8611.mjs
   │     │  │  ├── Ascensione_c410f67e.mjs
   │     │  │  ├── astro-assets-services_7d1e1c5d.mjs
   │     │  │  ├── astro_7ef08452.mjs
   │     │  │  ├── Buisson ardent._650da6f6.mjs
   │     │  │  ├── Caparica_3a53d1ec.mjs
   │     │  │  ├── Crazy birds. Fori imperiali._da8d9113.mjs
   │     │  │  ├── Dancing for the Waves._f8ff0c2b.mjs
   │     │  │  ├── Fairy tale._4e04abb7.mjs
   │     │  │  ├── Flying low._84666aa6.mjs
   │     │  │  ├── generic_d7a5de42.mjs
   │     │  │  ├── i_c851ae49.mjs
   │     │  │  ├── index_22559615.mjs
   │     │  │  ├── Inseparables _8d1898a9.mjs
   │     │  │  ├── Into her thoughts_8ada9a1e.mjs
   │     │  │  ├── Intrighi notturni_9e2e3d28.mjs
   │     │  │  ├── Lepi dor._e7609301.mjs
   │     │  │  ├── Light yellow_f94431d2.mjs
   │     │  │  ├── Luna indaco_004cd3cf.mjs
   │     │  │  ├── pages
   │     │  │  │  ├── generic_b87fddb0.mjs
   │     │  │  │  ├── i_aa022a95.mjs
   │     │  │  │  └── index_15415b19.mjs
   │     │  │  ├── Running into the waves._754cf851.mjs
   │     │  │  ├── Speed_740edfe9.mjs
   │     │  │  ├── The boy_c13d302e.mjs
   │     │  │  ├── The bug._a590fbf1.mjs
   │     │  │  ├── The scream of the butterfly_ebc91f03.mjs
   │     │  │  ├── Wandering_69eb00e6.mjs
   │     │  │  ├── Windy_cc002e91.mjs
   │     │  │  └── Winter is coming._1b332461.mjs
   │     │  ├── entry.mjs
   │     │  ├── manifest_42a2d5fd.mjs
   │     │  └── renderers.mjs
   │     ├── package.json
   │     └── src
   │        └── pages
   │           └── i.astro

**Comment by mikhi32** [comment]
Found the problem, is compleatley unrelated to what i said here

**Comment by rafaelcacilhas** [comment]
[Im having an error](https://github.com/orgs/vercel/discussions/5501#discussioncomment-8305480) that might be related. Can you explain what was the problem?
